### DISCUSSION TITLE
Versioni LTS

### DISCUSSION THREAD

**Discussion by vittoriomigliore** [italia/ansc#965] [Supporto servizi cooperativi] [answered]
Buongiorno,
al fine di stabilizzare l'integrazione con ANSC mi chiedevo se fosse stata battezzata una versione (ad esempio la 100014 - 1.30.x) come LTS.

Cordiali saluti
Vittorio Migliore
Gruppo Finmatica | ADS

**Comment by asqueglia** [comment, marked as answer]
Buongiorno, le versioni del sistema rilasciate, al momento durano al più 3 mesi; da settembre estenderemo il tempo di retrocompatibilità a 6 mesi come richiesto da alcune software house; quindi non c'è una versione  LTS.
